tsconfig.json

```diff
@@ -5,25 +5,25 @@
     { "path": "./packages/linzjs-docker-command/tsconfig.json" },
     { "path": "./packages/linzjs-geojson/tsconfig.json" },
     { "path": "./packages/linzjs-metrics/tsconfig.json" },
-    { "path": "./packages/sprites/tsconfig.json" },
-    { "path": "./packages/geo/tsconfig.json" },
-    { "path": "./packages/tiler/tsconfig.json" },
-    { "path": "./packages/tiler-sharp/tsconfig.json" },
-    { "path": "./packages/cli-raster/tsconfig.json" },
+    { "path": "./packages/_infra/tsconfig.json" },
+    { "path": "./packages/attribution/tsconfig.json" },
+    { "path": "./packages/bathymetry/tsconfig.json" },
     { "path": "./packages/cli/tsconfig.json" },
     { "path": "./packages/cli-config/tsconfig.json" },
+    { "path": "./packages/cli-raster/tsconfig.json" },
     { "path": "./packages/cli-vector/tsconfig.json" },
-    { "path": "./packages/bathymetry/tsconfig.json" },
-    { "path": "./packages/_infra/tsconfig.json" },
     { "path": "./packages/config/tsconfig.json" },
     { "path": "./packages/config-loader/tsconfig.json" },
-    { "path": "./packages/shared/tsconfig.json" },
-    { "path": "./packages/lambda-tiler/tsconfig.json" },
-    { "path": "./packages/lambda-analytics/tsconfig.json" },
+    { "path": "./packages/geo/tsconfig.json" },
     { "path": "./packages/lambda-analytic-cloudfront/tsconfig.json" },
-    { "path": "./packages/attribution/tsconfig.json" },
+    { "path": "./packages/lambda-analytics/tsconfig.json" },
+    { "path": "./packages/lambda-tiler/tsconfig.json" },
     { "path": "./packages/landing/tsconfig.json" },
     { "path": "./packages/server/tsconfig.json" },
-    { "path": "./packages/smoke/tsconfig.json" }
+    { "path": "./packages/shared/tsconfig.json" },
+    { "path": "./packages/smoke/tsconfig.json" },
+    { "path": "./packages/sprites/tsconfig.json" },
+    { "path": "./packages/tiler/tsconfig.json" },
+    { "path": "./packages/tiler-sharp/tsconfig.json" }
```

Comment:
this ordering is a little weird, `lambda-tiler` needs `tiler` and `tiler-sharp` so the order doesnt matter here?

Reply:
It would sort of imply that something is maybe missing a project reference? what error do you get when you move it to the bottom?